tests/com/tony/server/MainTest.java

```diff
@@ -24,7 +24,7 @@ public void testCreateURIListHasAllURIFromDirectory() throws Exception {
 
     @Test
     public void testCreateCobSpecRouterAddsResponsesToRouter() throws Exception {
-        Router cobSpecRouter = Main.createCobSpecRouter(System.getProperty("user.dir") + "/public/");
+        Router cobSpecRouter = Main.createCobSpecRouter(System.getProperty("user.dir") + "/public/", new ArrayList());
```

Comment:
This is a nitpick but I prefer to name all arguments in tests. I don't know what new ArrayList() is representing here.